schemas/user/summary.yml

```diff
@@ -0,0 +1,33 @@
+properties:
+  id:
+    type: string
+    readOnly: true
+    description: |
+      The user's identifier (generated by the API)
+
+  url:
+    type: string
+    readOnly: true
+    description: |
+      API url for accessing the user description.
+
+  email:
+    type: string
+    format: email
+    readOnly: true
+    description: |
+      The email address used when authenticating the user.
```

Comment:
An email address does give a way to contact someone if needed, so it is still useful. I think admins will also appreciate there being a way for users to reset their own passwords.

I think I'm fine with an email address as a username.